index.html

```diff
@@ -51,19 +54,316 @@ <h1>
                     <li><a href="#"><img src="assets/Vector4.png" alt=""/></a></li>
                     
                 </ul>
-
+ 
     
                 
     
     
             </div>
 
-        </section>
+        </section>     
+    </section>
+    <section class="recent-works">
+      <h1>My Recent Works</h1>
+      <div class="line-rule"></div>
+
+      <div class="img-placeholder">
+        <img src="assets/img-placeholder.png" alt=""/>
+      </div>
+
+      <div class="post">
+        <h2>Multi-Post Stories</h2>
+      </div>
+      <div class="content">
+        <p>
+          A daily selection of privately personalized reads; 
+          no accounts or sign-ups required. 
+          has been the industry's standard dummy text ever since the 1500s, 
+          when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+      <div class="tag">
+        <div class="tag-1">
+          <span>Css</span>
+        </div>
+        <div class="tag-2">
+          <span>Html</span>
+        </div>
+        <div class="tag-3">
+          <span>bootstrap</span>
+        </div>
+        <div class="tag-4">
+          <span>Ruby</span>
+        </div>
+      </div>
+      <div class="project-btn">
+        <h4>See Project</h4>
+      </div>      
+      <section class="projects">
 
         
+        <h1>Profesional Art Printing Data</h1>
+        <small>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required. 
+          has been the industry's standard
+        </small>
+        <div class="second-tags">
+         <div class="second-tag">
+          <span>Html</span>
+         </div>
+         <div class="second-tag2">
+          <span>bootstrap</span>
+         </div>
+         <div class="second-tag3">
+          <span>Ruby</span>
+         </div>
+        </div>
+
+        <div class="action">
+          <h3>See Project</h3>
+        </div>
+
+
+
+
         
 
+      </section>
+      <section class="projects">
+
         
+        <h1>Profesional Art Printing Data</h1>
+        <small>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required. 
+          has been the industry's standard
+        </small>
+        <div class="third-tags">
+          <div class="third-tag">
+           <span>Html</span>
+          </div>
+          <div class="third-tag2">
+           <span>bootstrap</span>
+          </div>
+          <div class="third-tag3">
+           <span>Ruby</span>
+          </div>
+         </div>
+
+        <div class="action">
+          <h3>See Project</h3>
+        </div>
```

Comment:
- [ ] [OPTIONAL] The `See Project` text should use a `button` or an `a` tag preferably a `button` tag so `button` interactions can be added 🙏